File: may_22_tabular/R/baseline.R
#just a simple baseline model
library(tidyverse)

trn <- read_csv(here::here("may_22_tabular/data/train.csv"))
tst <- read_csv(here::here("may_22_tabular/data/test.csv"))

pred <- sum(trn$target) / nrow(trn)

sub <- tibble(
    id = tst$id,
    target = pred
)

write_csv(sub, here::here("may_22_tabular/submissions/baseline_sub.csv"))

File: playground_series/s3e1/jl/baseline.jl
using CSV
using DataFrames
using Statistics

trn = CSV.read("./playground_series/s3e1/data/train.csv", DataFrame)
tst = CSV.read("./playground_series/s3e1/data/test.csv", DataFrame)

#take a peek at dimensions
size(trn)

pred = mean(trn.:MedHouseVal)

sub = DataFrame(
    id = tst.:id,
    MedHouseVal = pred
)

#write out
CSV.write("./playground_series/s3e1/submissions/baseline_mean.csv", sub)